ResearchKit/Charts/ORKFloatRange.h

```diff
@@ -37,52 +38,63 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /**
- The `ORKRangedPoint` class represents a ranged point used in a graph plot.
+ The `ORKFloatRange` class represents a float range. It can be used in graph chart plots to draw a
+ value ranges.
  */
 ORK_CLASS_AVAILABLE
-@interface ORKRangedPoint : NSObject
+@interface ORKFloatRange : NSObject
```

Comment:
Good point, I can rename it. And also `ORKFloatStack` to `ORKValueStack`(used by `ORKBarGraphChart` in the other PR).

Also, I was wondering, do you think that using `double` (or `float`?) instead of `CGFloat` would be better? As to make it more agnostic, given that these could be potentially used in other contexts that are unrelated to *CoreGraphics*.